components/scream/src/physics/share/scream_trcmix.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef SCREAM_TRCMIX_HPP
+#define SCREAM_TRCMIX_HPP
+
+#include "share/scream_types.hpp"
+
+//
+// Bridge function to call trcmix from CXX
+//
+
+namespace scream {
+namespace physics {
+
+void trcmix(const char* name, Int ncol, Int pcols, Int pver, Real* clat, Real* pmid, Real* q);
```

Comment:
Here is the API from SCREAM's cxx point of view. Do you want the arrays to be raw pointers, as they are here, or Kokkos Views?

Reply:
I'm not sure it matters. I'm imagining this routine being called from `RRTMGPRadiation::initialize_impl` to populate the gas concentrations stored in the Field Manager, so maybe it makes sense to have `trcmix` accept the kokkos views directly?